src/fmu/dataio/scripts/create_case_metadata.py

```diff
@@ -13,7 +13,13 @@
 import logging
 from pathlib import Path
 
+try:
+    import ert_shared
+except ModuleNotFoundError:
+    import ert.shared as ert_shared  # noqa
+
 from ert_shared.plugins.plugin_manager import hook_implementation  # type: ignore
+
```

Comment:
This will not work as importing ert.shared as ert_shared and attempting to re-import from that module will fail. I would just try to import `hook_implementation` in both branches. 
```suggestion
try:
    from ert_shared.plugins.plugin_manager import hook_implementation  # type: ignore
except ModuleNotFoundError:
    from ert.shared.plugins.plugin_manager import hook_implementation

```